torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -669,7 +662,7 @@ def _run_param_init_fn():
         for n, p in self.named_parameters():
             if p not in ignored_params and not isinstance(p, FlatParameter):
                 raise RuntimeError(
-                    f"found unsharded parameter: {n} ; {p.size()} {p.__class__}"
+                    f"found unflattened parameter: {n} ; {p.size()} {p.__class__}"
```

Comment:
hmm, why changing `unsharded` to `unflattened`? I thought we are checking all parameters are sharded here, no?

Reply:
for world_size=1 or NO_SHARD strategy, parameters are not sharded.